fesvr/memif.h

```diff
@@ -19,6 +25,11 @@ class chunked_memif_t
 
   virtual size_t chunk_align() = 0;
   virtual size_t chunk_max_size() = 0;
+
+  virtual void set_target_endianness(memif_endianness_t endianness) {}
+  virtual memif_endianness_t get_target_endianness() const {
+    return memif_endianness_undecided;
+  }
```

Comment:
I think the part may be optimized the target_big_endian part to use constant to help compiler and remove ifdef in overloading functions.

Reply:
Not sure what you mean here.  `target_big_endian` is in `mmu_t`, which is a concrete class and a completely separate API from `memif_t`.